chalice/utils.py

```diff
@@ -4,7 +4,24 @@
 
 from typing import IO, Dict, Any  # noqa
 
+from chalice import __version__ as chalice_version
 from chalice.constants import WELCOME_PROMPT
+from chalice.config import Config  # noqa
+
+
+def get_application_tags(config):
```

Comment:
Capturing offline discussion: I'm fine with moving this over to the `Config` object.